wdl/MainVcfQc.wdl

```diff
@@ -0,0 +1,845 @@
+version 1.0
+
+# Author: Ryan Collins <rlcollins@g.harvard.edu>
+
+# Note: this WDL has been customized specifically for the gnomAD-SV v3 callset
+# Some components of this WDL will not be generalizable for most cohorts
+
+import "ShardedQcCollection.wdl" as ShardedQcCollection
+import "CollectQcPerSample.wdl" as CollectQcPerSample
+import "ShardedCohortBenchmarking.wdl" as CohortExternalBenchmark
+import "PerSampleExternalBenchmark.wdl" as PerSampleExternalBenchmark
+import "Tasks0506.wdl" as MiniTasks
+import "Utils.wdl" as Utils
+
+# Main workflow to perform comprehensive quality control (QC) on
+# an SV VCF output by GATK-SV
+workflow MasterVcfQc {
+  input {
+    Array[File] vcfs
+    Array[File] vcf_idxs
+    Boolean vcf_format_has_cn = true
+    File? ped_file
+    File? list_of_samples_to_include
+    Int max_trios = 1000
+    String prefix
+    Int sv_per_shard
+    Int samples_per_shard
+    Array[Array[String]]? site_level_comparison_datasets    # Array of two-element arrays, one per dataset, each of format [prefix, gs:// path to directory with one BED per population]
```

Comment:
The site-level and sample-level datasets will not be a matched set, but we could split the `Array[Array[String]]` for each into two `Array[String]` inputs, although then I would worry about users making mistakes in the ordering of the two arrays. `Array[Array[]]` helps to avoid that (or `Pair()`, although I seem to recall there was some funkiness with the behavior of `Pair` when I tried to implement it here)

Reply:
For the sake of Terra compatibility, can we make this input a table `File site_level_comparison_datasets_tsv` and use `read_tsv()`?